tests/prow_ci.sh

```diff
@@ -30,3 +30,163 @@ override_rosacli_build () {
   # go back to previous dir
   cd -
 }
+
+# configure_aws will setup aws credentials for the CI job run
+# It requires two parameters 
+# the first one for aws credential file path which is required
+# the second one for aws region which is required
+# the third one for second aws cred info for shared vpc scenario which is optional
+# usage: configure_aws <aws file path> <region> <shared vpc credential file>
+configure_aws () {
+  # configure aws region
+  if [[ -z "$1"]] || [[ -z "$2" ]]; then
+    echo "ERROR: aws credential file path and region is required. Please call command like $ configure_aws <aws path> <region>"
+    exit 1
+  fi
+  echo "[CI] Configured AWS region to ${REGION}"
+  # configure aws credetials
+  awscred=$1
+  if [[ -f "${awscred}" ]]; then
+    export AWS_SHARED_CREDENTIALS_FILE="${awscred}"
+    export AWS_DEFAULT_REGION="$2"
+  else
+    echo "ERROR: aws credential file $1 doesn't exist"
+    exit 1
+  fi
+
+  # configure shared vpc credentials
+  if [[ ! -z "$3" ]]; then
```

Comment:
Another method `aws_config_shared_vpc` is better.

Reply:
Let's use `configure_aws` since it's an action

Reply:
Moved it out.